pvlib/iotools/bsrn.py

```diff
@@ -0,0 +1,141 @@
+"""Functions to read data from the Baseline Surface Radiation Network (BSRN).
+.. codeauthor:: Adam R. Jensen<adam-r-j@hotmail.com>
+"""
+
+import pandas as pd
+import gzip
+import os
+
+COL_SPECS = [(0, 3), (4, 9), (10, 16), (17, 22), (23, 27), (28, 32), (33, 39),
+             (40, 45), (46, 50), (51, 55), (56, 64), (65, 70), (71, 75)]
+
+BSRN_COLUMNS = ['day', 'minute',
+                'ghi', 'ghi_std', 'ghi_min', 'ghi_max',
+                'dni', 'dni_std', 'dni_min', 'dni_max',
+                'empty', 'empty', 'empty', 'empty', 'empty',
+                'dhi', 'dhi_std', 'dhi_min', 'dhi_max',
+                'lwd', 'lwd_std', 'lwd_min', 'lwd_max',
+                'air_temperature', 'relative_humidity', 'pressure']
+
+
+def read_bsrn(filename):
+    """
+    Read a BSRN station-to-archive file into a DataFrame.
+
+    The BSRN (Baseline Surface Radiation Network) is a world wide network
+    of high-quality solar radiation monitoring stations as described in [1]_.
+    The function only parses the basic measurements (LR0100), which include
+    global, diffuse, direct and downwelling long-wave radiation [2]_. Future
+    updates may include parsing of additional data and meta-data.
+
+    Required username and password are easily obtainable by writing an email to
+    Amelie Driemel (Amelie.Driemel@awi.de) [3]_ on condition that users follow
+    BSRN's Data Release Guidelines [4]_.

```

Comment:
```suggestion
    Required username and password are easily obtainable as described in the
    BSRN's Data Release Guidelines [4]_.

```

I wouldn't hardcode someone's email into the function docstring. A link to their page should be enough.